recipes/giotto-learn/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "giotto-learn" %}
+{% set version = "0.1a" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/{{ name }}/{{ name }}.git
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script:
+    - git submodule update --init --recursive
+    - cmake -G "Visual Studio 15 2017"  # [win]
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [not win]
+    - vs2017_win-64  # [win]
+    - cmake
+    - boost >=1.56.*
+    - git
+    - setuptools
+  host:
+    - python
+    - setuptools
+    - pip
+    - numpy >=1.17.0
+    - scipy
+    - scikit-learn >=0.21.3
+  run:
+    - python
+    - numpy >=1.17.0
+    - scipy
+    - scikit-learn >=0.21.3
+
+test:
+  requires:
+    - pytest >=3.3.0
+  commands:
+    - pytest -h
```

Comment:
this command should be the pytest to test this project

Reply:
I would recommend adding the ``import`` section here
Please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=test#the-recipe-meta-yaml
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=test#test
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=test#testing-importing

Reply:
import section added and working!

Reply:
All test are now passing (I removed the -h option)!